src/20241014_AaveV3Ethereum_AaveCertoraContinuousSecurityServices/AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014.t.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AggregatorInterface} from 'aave-v3-origin/contracts/dependencies/chainlink/AggregatorInterface.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {IERC20Metadata} from 'solidity-utils/contracts/oz-common/interfaces/IERC20Metadata.sol';
+import {IStreamable} from 'aave-address-book/common/IStreamable.sol';
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/src/ProtocolV3TestBase.sol';
+import {AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014} from './AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014.sol';
+
+/**
+ * @dev Test for AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014
+ * command: FOUNDRY_PROFILE=mainnet forge test --match-path=src/20241014_AaveV3Ethereum_AaveCertoraContinuousSecurityServices/AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014.t.sol -vv
+ */
+contract AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014_Test is ProtocolV3TestBase {
+  AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014 internal proposal;
+  uint256 public constant BLOCKS_IN_FOUR_HOURS = (4 * 3600) / 12; //number of blocks in 4h
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 20965854);
+    proposal = new AaveV3Ethereum_AaveCertoraContinuousSecurityServices_20241014();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
```

Comment:
actually we are missing e2e tests which does the snapshot diff too

Reply:
I removed the e2e as we don't touch the pool, want me to add them back or just remove the comment ?